runner.py

```diff
@@ -29,13 +29,14 @@
     parser = argparse.ArgumentParser()
     parser.add_argument('--configs', metavar='ConfigPath', type=str,
                         default='configs/config_example.json',
-                        help='Path to configuration files')
+                        help='Path to configuration files or '
+                             'Path to directory which contains configuration files')
     parser.add_argument('--dummy-run', default=False, action='store_true',
                         help='Run configuration parser and datasets generation '
                              'without benchmarks running')
     parser.add_argument('--no-intel-optimized', default=False, action='store_true',
                         help='Use no intel optimized version. '
```

Comment:
"Use no intel optimized version." -- this sentence is unclear

Reply:
What do you think about "Use libraries without intel optimizations" ?

Reply:
- `Use libraries` -- does it mean that this is not only about scikit-learn but other libraries as well?
- `intel` -> `Intel`

other than that, this phrasing is ok

Reply:
Yes, this benchmark can run with other libraries besides scikit-learn